R/dependencies.R

```diff
@@ -512,49 +516,49 @@ renv_dependencies_discover_renv_lock <- function(path) {
 renv_dependencies_discover_description <- function(path,
                                                    fields = NULL,
                                                    subdir = NULL,
-                                                   project = NULL)
+                                                   project = NULL,
+                                                   is_dependency = FALSE)
```

Comment:
How do you feel about using `fields = "most"` in the callers, and avoiding the addition of a new parameter here? See also https://github.com/rstudio/renv/blob/557433bafa4677221a1461588290dc71421bc664/R/description.R#L168-L196.

Reply:
Oops, I guess I mean "strong", not "most".

Reply:
Oh yeah, that works much better. That also required a change to `renv_project_remotes()`, but I think for the better — now no-arg `install()` will respect the active profile.